test/alignak_test.py

```diff
@@ -131,127 +132,94 @@ class Pluginconf(object):
 class AlignakTest(unittest.TestCase):
 
     time_hacker = TimeHacker()
+    maxDiff = None
```

Comment:
It's uses by nosetests to display a complete diff between 2 dicts in case self.assertEqual() not ok